doc/radosgw/cloud-restore.rst

```diff
@@ -142,121 +159,133 @@ For example:
 
-Example,
-
-::
+Example:
 
+```
   $ aws s3api head-object --key doc1.rtf --bucket bucket1 ....
+```
 
+The ``radosgw-admin`` CLI can be used to check restoration status and other
+details.
 
-In addition, ``radosgw-admin`` CLI can be used to check the restoration status and other details on the RGW server.
-
-Example,
+Example:
 
-::
-  
+```  
  $ radosgw-admin object stat --bucket bucket1 --object doc1.rtf
+```
 
 
 Restored Object Properties
 --------------------------
 
-
 Storage
 ~~~~~~
-The objects are restored to the storage class configured for ``restore_storage_class`` tier-config option. However, as per `AWS S3 RestoreObject <https://docs.aws.amazon.com/cli/latest/reference/s3api/restore-object.html>`__ the storage class of restored objects should remain unchanged. Therefore, for temporary copies, the x-amz-storage-class will continue to reflect the original cloud-tier storage class.
+Objects are restored to the storage class configured via ``restore_storage_class``
+in the tier-config. However, as
+per `<https://docs.aws.amazon.com/cli/latest/reference/s3api/restore-object.html>`
+the storage class of restored objects should remain unchanged. Therefore, for
+temporary copies, the ```x-amz-storage-class``` will continue to reflect the
+original cloud-tier storage class.
 
 
 mtime
 ~~~~
-The `mtime` of the transitioned and restored objects should remain unchanged.
+The ``mtime`` of the transitioned and restored objects should remain unchanged.
 
 
 Lifecycle
 ~~~~~~~~
-`Temporary` copies are not subjected to any further transition to the cloud. However (as is the case with `cloud-transitioned objects`) they can be deleted via regular LC expiration rules or via external S3 Delete request.
-`Permanent` copies are treated as any regular objects and are subjected to any LC rules applicable.
+``Temporary`` copies are not subject to transition to the cloud. However, as is the
+case with cloud-transitioned objects, they can be deleted via regular LC (Life Cycle)
+expiration rules or an external S3 ``delete`` request.
+
+``Permanent`` copies are treated as regular objects and are subject to applicable LC
+policies.
 
 
 Replication
 ~~~~~~~~~~
-`Temporary` copies are not replicated and will be retained only on the zones the restore request is initiated on.
-`Permanent` copies are replicated like other regular objects.
+``Temporary`` copies are not replicated and will be retained only by the zone
+on which the restore request is initiated.
+
+``Permanent`` copies are replicated like other regular objects.
 
 
 Versioned Objects
 ~~~~~~~~~~~~~~~~
-For versioned objects, if an object has been `cloud-transitioned`, it would be in a non-current state. After a restore, the same non-current object will be updated with the downloaded data, and its HEAD object will be modified accordingly.
+For versioned objects, if an object has been cloud-transitioned, it is in a
+non-current state. After a restore, the same non-current object will be
```

Comment:
s/restore/restoration/

I suppose this kind of thing is likely to get worse and worse as time goes on and the various parts of speech are used more and more incorrectly, but it really does make the English look like it was written by someone with a limited understanding of the syntactical functions of the various parts of speech.